src/stories/sidenav/fd-side-nav.stories.ts

```diff
@@ -0,0 +1,172 @@
+                        <a fd-nested-list-link>
+                            <span *ngIf="!mainTextOnlyVar" fd-nested-list-icon [glyph]="icon"></span>
+                            <span fd-nested-list-title>{{textValue1}}</span>
+                        </a>
+                    </li>
+                    <li fd-nested-list-item>
+                        <a fd-nested-list-link>
+                            <span *ngIf="!mainTextOnlyVar" fd-nested-list-icon [glyph]="icon"></span>
+                            <span fd-nested-list-title>{{textValue1}}</span>
+                        </a>
+                    </li>
+                    <li fd-nested-list-item>
+                        <a fd-nested-list-link>
+                            <span *ngIf="!mainTextOnlyVar" fd-nested-list-icon [glyph]="icon"></span>
+                            <span fd-nested-list-title>{{textValue1}}</span>
+                        </a>
+                        <ul fd-nested-list [textOnly]="mainSubTextOnlyVar">
+                            <li fd-nested-list-item>
+                                <a fd-nested-list-link>
+                                    <span *ngIf="!mainSubTextOnlyVar" fd-nested-list-icon [glyph]="iconSecondary"></span>
+                                    <span fd-nested-list-title>{{textValue1}}</span>
+                                </a>
+                                <a fd-nested-list-link>
+                                    <span *ngIf="!mainSubTextOnlyVar" fd-nested-list-icon [glyph]="iconSecondary"></span>
+                                    <span fd-nested-list-title>{{textValue1}}</span>
+                                </a>
+                            </li>
+                        </ul>
+                    </li>
+                </ul>
+            </div>
+        
+            <div fd-side-nav-utility>
+                <ul fd-nested-list 
+                        [compact]="utilityCompactVar" 
+                        [textOnly]="utilityTextOnlyVar">
+                    <li fd-nested-list-item>
+                        <a fd-nested-list-link>
+                            <span *ngIf="!utilityTextOnlyVar" fd-nested-list-icon [glyph]="iconUtility"></span>  
+                            <span fd-nested-list-title>{{textValue1}}</span>
+                        </a>
+                    </li>
+                    <li fd-nested-list-item>
+                        <a fd-nested-list-link>
+                            <span *ngIf="!utilityTextOnlyVar" fd-nested-list-icon [glyph]="iconUtility"></span>
+                            <span fd-nested-list-title>{{textValue1}}</span>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </fd-side-nav>
+    `,
+    props: {
+        mainCompactVar: boolean('Main Secion in Compact Mode', false),
+        mainTextOnlyVar: boolean('Main Section Text Only', false),
+        mainSubTextOnlyVar: boolean('Main Sub Section Text Only', false),
```

Comment:
Let's remove `mainSubTextOnlyVar`. It should always be set to false

![image](https://user-images.githubusercontent.com/26483208/81055455-fc8e2980-8ec8-11ea-91db-efb01940480d.png)

Reply:
fixed